backend/routes/auth_routes.py

```diff
@@ -0,0 +1,111 @@
+from uuid import uuid4
+from jose import jwt,JWTError
```

Comment:
e5f2e6fa5dbc63f1775d52a333f1e332b662daf1

@Rhythm1710 The dependency is still jose. I think there might be some confusion while handling the commit. Could you please re-verify ?

Reply:
Yes Sure!

Reply:
Actually I have changed the jose library and I am currently using python-jose but its with the same name.
![image](https://github.com/Nactore-Org/Nacto-Care/assets/65298117/33ee4e46-5861-4ed4-bc36-37d207901977)

Reply:
If the issue still persists let me know. Please tell is there any changes other than this library.

Actually both jose and python-jose jose is used as from jose import ...
That's creating confusion.

Reply:
Python Goodness !! No worries, I think others won't face the issue if we include in requirements.txt for now to test I have manually installed python-jose in place of jose and it's running. Althought it gives Internal Server 500 probably due to schema mapping.

I will check in more details and revert back tomorrow.

Reply:
Ok Sure!

Reply:
Also I have created some more routes in Patient's routes.Shoud I push that now or wait till tomorrow?

Reply:
I think the error you are facing is because of change in the models. There was no password column in that I have added that. You can delete the database or create new column. Then it will run fine.

Reply:
I can create another issue to handle such problems by using the alembic. Should I do that?